extensions/replication/replicationStatusProcessor/ReplicationStatusProcessor.js

```diff
@@ -336,7 +334,6 @@ class ReplicationStatusProcessor {
             sourceHTTPAgent: this.sourceHTTPAgent,
             vaultclientCache: this.vaultclientCache,
             gcProducer: this._gcProducer,
-            mProducer: this._mProducer,
             statsClient: this._statsClient,
```

Comment:
I think `_statsClient` is also part of the "Redis metrics". To be double checked, but i think it also needs to be removed.

Reply:
Same in other files.